### ISSUE TITLE
Traning with 2 GPUs and the results were weird?

### ISSUE THREAD

**Issue by wzan0001** [bowenc0221/panoptic-deeplab#34]
Thank you for sharing the amazing job!
I have 2 2020ti gups for training, and the backbone is mobilenet. cause of the gpu's ram, only can set IMS_PER_BATCH with 2 and MAX_ITER keep 90000. Then, i get weird results of dome.py, all target(car/people) in panoptic_pred.png is black and background is color, then i check the instance.png, it's black too.  So i need more iterations? some suggestion about learning rate?

**Comment by bowenc0221** [comment]
I would suggest you play with these configs:
```
DATASET:
  CROP_SIZE: (1025, 2049)
SOLVER:
  BASE_LR: 0.0001
TRAIN:
  IMS_PER_BATCH: 16
  MAX_ITER: 90000
```

1. Choose a smaller crop size (e.g. to (513, 1025)) to fit a larger batch size. A batch size of 2 may be too small for BatchNorm.
2. When you change crop size and batch size, you will need to change both lr and max_iter. Unfortunately I also don't know what is a good choice for your case, you will need to figure it out by yourself.
3. Please check the debug images during training. Those are model predictions on the training image. If your model cannot fit training images well, there must be something wrong with your training parameters.

**Comment by wzan0001** [comment]
Thank you for your reply
I check the debug images like this:
[https://github.com/wzan0001/image/blob/master/11/debug_batch_outputs_89899.png](url)
panoptic_pred image :
[https://github.com/wzan0001/image/blob/master/11/panoptic_pred_5.png](url)
and the semantic segmentation :
[https://github.com/wzan0001/image/blob/master/11/semantic_pred_3.png](url)

Need i modify the CENTER_THRESHOLD?

**Comment by bowenc0221** [comment]
Cannot see your images.
I think it should not be related to CENTER_THRESHOLD.
You can also try setting `TEST.DEBUG` to `True`.

**Comment by wzan0001** [comment]
Thank you for your reply,
debug_batch output like this:
![debug_batch_outputs_89899](https://user-images.githubusercontent.com/46546215/89375044-6d8aea80-d71f-11ea-9dea-dfad6e0c6a9e.png)

panoptic output likes:
![panoptic_pred_5](https://user-images.githubusercontent.com/46546215/89375081-84c9d800-d71f-11ea-8a7e-b77fd27d2051.png)

and the semantic likes:
![semantic_pred_3](https://user-images.githubusercontent.com/46546215/89375096-8e534000-d71f-11ea-99cf-16dea33e0040.png)

need i modify the CENTER_THRESHOLD?

**Comment by wzan0001** [comment]
Now, I set TEST.DEBUG to True, and the debug_test.png likes this:
![debug_test_outputs_3](https://user-images.githubusercontent.com/46546215/89376773-1b4bc880-d723-11ea-8db4-242d84490d72.png)

**Comment by bowenc0221** [comment]
The intermediate outputs look good! I don't think it is CENTER_THRESHOLD. But you can check by setting CENTER_THRESHOLD to 0.

Did you modify any code?

**Comment by wzan0001** [comment]
I haven't modified any other code, and I downloud the mobilenet pretrained model, it's get correct result.
I try CENTER_THRESHOLD to 0.0, but no effect. And if use the pretrained model with CENTER_THRESHOLD=0, it's also get black?

**Comment by bowenc0221** [comment]
It should not be black, black is VOID region.

**Comment by wzan0001** [comment]
so  I will reset crop_size and training it again, tanks for your reply. I will give you feedback after training.

**Comment by wzan0001** [comment]
Hi, I set crop_size with (513, 1025), batch size with 10 and MAX_ITER keep 90000,  it output the right result! The PQ is 0.54, I think it could be improved.  
But i am not sure the reason for previous error.

**Comment by bowenc0221** [comment]
Great! Probably due to some weird behavior of BatchNorm.
